tests/integ/modin/conftest.py

```diff
@@ -344,8 +344,8 @@ def float_string_frame():
     """
     df = pandas.DataFrame(
         np.random.default_rng(2).standard_normal((30, 4)),
-        index=pd.Index([f"foo_{i}" for i in range(30)], dtype=object),
-        columns=pd.Index(list("ABCD"), dtype=object),
+        index=pandas.Index([f"foo_{i}" for i in range(30)], dtype=object),
```

Comment:
i see, so this fixture actually creates a native pandas object . @sfc-gh-vbudati it is not related to this fix, but how does our pd.DataFrame inteface work with the pd.Index class we have today? does it work or fail? can you create a ticket to check this?

Reply:
yeah, here's the ticket to investigate: https://snowflakecomputing.atlassian.net/browse/SNOW-1542799